src/features/slackapi/slackapi-utils.ts

```diff
@@ -207,10 +219,19 @@ Have a great week!
       const message = constructWeeklySummaryMessage(userData, weekStartDate.toISODate(), weekEndDate.toISODate());
 
       if (!isAway && !firstDayBack) {
-        messageResults.push({
-          message: message,
-          response: await sendMessage(slackId, message.message)
-        });
+        if (!listOfIds) {
```

Comment:
Coveralls checks are failing, we can see from running the tests that the listOfIds is not being found in the postWeeklyMessageToUsers function. 
This is because you are declaring the listOfIds variable in the postDailyMessage function, so the scope does not extend to postWeeklyMessageToUsers. 
You could declare the listOfIds at the top of the file beneath the client declaration and then it should reference correctly in both functions and all test should pass.

Once this is done you should also check the functions work as intended then there is no listOfIds passed in, by invoking the function locally and replacing the slack sendMessage functions with logs.